tests/jest.setup.js

```diff
@@ -175,6 +175,30 @@ jest.mock( "components/Camera/FadeInOutView", () => {
   return jest.fn( ( ) => React.createElement( View, null ) );
 } );
 
+// Mock @react-navigation/bottom-tabs to disable animations in Jest tests
+// This prevents the act() warnings caused by fade animations triggering state updates
+jest.mock( "@react-navigation/bottom-tabs", () => {
```

Comment:
That looks awfully complicated but I am okay if it works for us. Commenting this out makes one test fail so something is needed here.
react-navigation recommends to not mock but use fake timers I think. Have you tried something along the lines of this? https://reactnavigation.org/docs/testing/#navigation-between-tabs